### PR TITLE
Dashboard skeleton

### PR DESCRIPTION
**Pull request by mattchx**
Screen Shot 2021-01-12 at 21 18 00

Note: This is a draft and is not fully complete.

**Components included:**
NavBar(Links + Profile info)
Title
SelectTabs
  EventTypesTab(contains profile info, CreateNewType Btn, and Event Cards)
  ScheduledEventTab( Empty and could be disabled)
Getting Started Button

**Issues needed to be handled:**
- add 3 icons: plus, clock, and setting/edit
- convert card component to accept props for: color bar, length of meeting, and number of participants.
- used a mix of Box(flexbox) and grid. May switch all to grid.
- added secondary color(Orange) to theme.js, I have yet to implement in all components
- must increase responsive design for different screen sizes & add navbar for mobile
- need to replace avatar with photo, adjust white space, font-sizes, and colors

_I appreciate any and all feedback!_

### CODE REVIEW

File: client/src/components/Header/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+export default function NavBar () {
+  const classes = useStyles();
+  const preventDefault = (event) => event.preventDefault();
+
+return (
+  <AppBar className={classes.appBar} position='static'>
+        <Toolbar>
+          <Typography variant='h6' className={classes.title}>
+            <span className={classes.orangeText}>calend</span>App
```
Comment (dproc96) [inline client/src/components/Header/NavBar.js]: In place of this you could use the logo image that was provided, would be easier to match the mocks that way

**Review by dproc96 — COMMENTED** [review]
So far this looks good, pending the changes you mentioned already. There are two things I noticed if you want to even closer match the mocks: the text in the navbar could be a little smaller and more spaced and the user's name in event types could be bolded.

These are nitpicks though, great work so far!

File: client/src/components/Header/NavBar.js (hunk 1 above)
Comment (mattchx) [reply client/src/components/Header/NavBar.js]: Thanks Dan! I'm on it.

**Comment by mattchx** [comment]
Screen Shot 2021-01-13 at 21 41 44

I'm happy with how the elements all came together. Let me know if there are any issues!
Edit: I was not able to soften the NavBar shadow nor did I align the User image under event types.

File: client/src/components/Header/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+export default function NavBar() {
+  const classes = useStyles();
+  const preventDefault = (event) => event.preventDefault();
+
+  return (
+    <AppBar className={classes.root} position='static'>
+      <Toolbar>
+        <span className={classes.title}>
+          <img width='110rem' src={Logo} />
```
Comment (dproc96) [inline client/src/components/Header/NavBar.js]: Be sure to add an alt attribute for these images

**Review by dproc96 — APPROVED** [review]
Be sure to add the alt attributes to your images before merging, but once you've done that this looks good to go!

**Pull request merged**